arch/cpu/cc26x0-cc13x0/rf-core/prop-mode.c

```diff
@@ -816,7 +827,6 @@ release_data_entry(void)
 {
   rfc_dataEntryGeneral_t *entry = (rfc_dataEntryGeneral_t *)rx_read_entry;
   uint8_t *data_ptr = &entry->data;
-  int_master_status_t interrupt_status;
```

Comment:
I suppose this critical section was there to make the read-and-set operation of `rf_core_rx_is_full` atomic. As the variable is now removed, it could be safe to remove the critical section as well.

Reply:
What critical section? I might be missing something but I don't see it...

Reply:
It was more of a comment to myself, really, to remember that I checked this code and it seemed safe.

Previously there was a `critical_enter()` / `critical_exit()` block, your PR removes these guards.